### PR TITLE
Add desktop screen

### PR DESCRIPTION
**Pull request by rotimiazeez**
#Capstone Project:  Leehar's Fashion Week👗

> A capstone project built based on a planned fashion week event with inspiration from the Cindy Shin conference website. 💪🏽  

Additional description of the project and its features. ➕ 

- Use semantic HTML tags.
- Apply best practices in HTML code.
- Use CSS selectors correctly.
- Use CSS box model.
- Use Flexbox to place elements on the page.
- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
- Use GitHub Pages to deploy web pages.
- Apply JavaScript best practices and language style guides in code.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Use objects to store and access data.
- Communicate technical concepts to other technical people.

## Built With ❤️ 

- HTML (HTML5 semantic tags, `ul`, `li`,  `button`)
- CSS ( `Flexbox`, `Grid`, `background-images` etc. )
- Javascript (Dom manipulation)

## Deployed Preview 🚀 
[Live Preview](https://deploy-preview-4--rotimiazeez-portfolio.netlify.app/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -258,14 +429,23 @@ <h6 class="margin-top-28 font-size-28 line-height-24 margin-left-8 pad-t-8 black
     </section>
 
     <footer class="background-white p-rel pad-top-4em">
```
Comment (rloterh) [inline index.html:431]: - The footer area has extra gray space below it. We actually want the design to look as close much to the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Azeez, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- Highly organized/clean code
- Good ReadMe
- No linter errors


## Required Changes ♻️
- [x] _Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rotimiazeez** [comment]
Hi @rloterh 🖐🏽,   thanks for the review 👍🏽. I totally missed the space at the bottom of the page, but it's fixed now. 💪🏽

File: css/style.css [hunk 2]
```diff
@@ -689,3 +717,284 @@ a {
   left: 33%;
   top: 99%;
 }
+
+@media only screen and (min-width: 768px) {
```
[the remark this answers is not in the document]
Comment (rotimiazeez) [reply css/style.css]: 👍

**Review by AshakaE — APPROVED** [review]
### Hi @rotimiazeez  🙋 , 

# STATUS APPROVED ✔️ 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

- Please try as much as possible to mimic the font weights and proportionality of elements from the Figma template. Thank you!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by rotimiazeez** [comment]
Hi @AshakaE🙋‍♀️,
Good to see you on my PR again✌.
Thanks for the Approval, for the font I changed some of them because the Figma font did not make it look like what's on the template.🤷‍♀️

**Pull request merged**